### PR TITLE
Remove old code using im_status

### PR DESCRIPTION
**Pull request by xdu-odoo**

### CODE REVIEW

**Comment by xdu-odoo** [comment]
And that's a green :ok_hand:

File: addons/mail/static/tests/mail_status_manager_tests.js [hunk 1]
```diff
@@ -1,190 +0,0 @@
...
-var Widget = require('web.Widget');
-
-var QWeb = core.qweb;
-
-QUnit.module('mail', {}, function () {
-QUnit.module('service', {}, function () {
-QUnit.module('Status manager', {
-    beforeEach: function () {
-        this.services = mailTestUtils.getMailServices(this);
-        this.timeoutMock = mailTestUtils.patchMailTimeouts();
```
Comment (alexkuhn) [inline addons/mail/static/tests/mail_status_manager_tests.js:17]: I think patch of mail timeouts is only used with fetch im status. If so, these test helpers should be removed as well

**Review by alexkuhn — CHANGES_REQUESTED** [review]

File: addons/mail/static/tests/mail_status_manager_tests.js (hunk 1 above)
Comment (xdu-odoo) [reply addons/mail/static/tests/mail_status_manager_tests.js:17]: I'll check that

**Comment by xdu-odoo** [comment]
Green again

File: addons/mail/static/tests/helpers/test_utils.js [hunk 2]
```diff
@@ -87,111 +87,6 @@ var MockMailService = Class.extend({
...
-    mailUtils.setTimeout = function (func, duration) {
-        duration = duration || 0;
-        var executeTime = currentTime + duration;
-        countTimeout++;
-        timeouts[countTimeout] = {
-            executeTime: executeTime,
-            func: func
-        };
-        return countTimeout;
-    };
```
Comment (alexkuhn) [inline addons/mail/static/tests/helpers/test_utils.js:113]: - both `mail.utils` helpers <`set`/`clear`>`Timeout()` no longer used elsewhere, so they should also be removed
- `mail.utils` no longer used in this file, so remove related `require()`

**Review by alexkuhn — CHANGES_REQUESTED** [review]

**Comment by xdu-odoo** [comment]
Greeeen

**Review by alexkuhn — APPROVED** [review]
squash then merge

**Pull request merged**